models/comments-model.js

```diff
@@ -0,0 +1,27 @@
+//REQUIRES
+const db = require("../db/connection.js");
+
+//FUNCTION
+function selectComments(article_id) {
+    const onlyDigitsRegex = /(^[0-9]+$)/
+    if (onlyDigitsRegex.test(article_id)) {
+            return db
+              .query("SELECT * FROM comments WHERE comments.article_id = $1 ORDER BY created_at DESC;", [article_id])
+                .then(({ rows }) => {
```

Comment:
You will need to consider that there are 2 reasons that rows could be empty:
1) article exists but had no comments (status 200)
2) article does not exist (status 404)